### PR TITLE
Fix restore from a Django ContentFile

### PR DESCRIPTION
**Pull request by dorianamouroux**
Read directly the file if it's a Django ContentFile (in-memory file).

### CODE REVIEW

File: dbbackup/db/base.py [hunk 1]
```diff
@@ -141,8 +143,13 @@ def run_command(self, command, stdin=None, env=None):
         full_env.update(self.env)
         full_env.update(env or {})
         try:
-            process = Popen(cmd, stdin=stdin, stdout=stdout, stderr=stderr,
-                            env=full_env)
+            if type(stdin) == ContentFile:
```
Comment (ZuluPro) [inline dbbackup/db/base.py]: Could you change `type(stdin) == ContentFile` to `isinstance(stdin, ContentFile)` ?

**Comment by ZuluPro** [comment]
Hello @dorianamouroux 
Thanks for this PR and sorry for my latency.

Could you just update the commit (not another one) with the comment ?

**Comment by ZuluPro** [comment]
Hello @dorianamouroux 
Thanks for this PR and sorry for my latency.

Could you just update the commit (not another one) with the comment ?

Just for curiosity, do you use DBBackup outside of its native mechanisms ?

**Comment by pkkid** [comment]
For what it's worth I hit this issue as well.  However, I also needed to include SFTPStorageFile in the tytpe check of stdin. I applied these changes as well as @ZuluPro's suggestion to the branch 'fix-restore' here:

https://github.com/django-dbbackup/django-dbbackup/tree/fix-restore

I'd love if we can get this merged, but I suspect we want a more generic type check than SFTPStorageFile.  I don't understand the issue enough to make it more of a catch all.

**Comment by pkkid** [comment]
I merged pull request #280 which has this change.

**Pull request closed without merging**